src/index.ts

```diff
@@ -104,7 +108,8 @@ export const DEFAULT_OPTIONS: Options = {
     useTabs: false
   },
   unreachableDefinitions: false,
-  unknownAny: true
+  unknownAny: true,
+  enableBigInt: false // True would be more sensible, but also a breaking change from older versions.
```

Comment:
Nit: Alphabetize please. Also no need for the comment.

```suggestion
  enableBigInt: false
```